### PR TITLE
chore: update steps for local dev setup

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Updates the steps for local dev setup

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: content/en/contribute/code/core/dev-environment.md [hunk 1]
```diff
@@ -149,11 +154,11 @@ npm run local-images
 
 After the `npm` command completes successfully, a `local-build` folder will be created in the root directory of your `cht-core` project.
 
-Confirm you have these four files in the `local-build` folder: `docker-compose.yml`, `cht-core.yml`, `cht-couchdb.yml` and `cht-couchdb-clustered.yml`
+Confirm you have these four files in the `local-build` folder: `cht-core.yml`, `cht-couchdb.yml` and `cht-couchdb-clustered.yml`
```
Comment (Benmuiruri) [inline content/en/contribute/code/core/dev-environment.md:157]: Confirm you have these three files in the `local-build` folder: `cht-core.yml`, `cht-couchdb.yml` and `cht-couchdb-clustered.yml`

**Review by Benmuiruri — CHANGES_REQUESTED** [review]

**Review by Benmuiruri — CHANGES_REQUESTED** [review]
Thanks for testing out and improving the docs. Just one small fix

File: content/en/contribute/code/core/dev-environment.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (latin-panda) [reply content/en/contribute/code/core/dev-environment.md:157]: Thanks!

**Comment by latin-panda** [comment]
@Benmuiruri can you please review?

**Review by Benmuiruri — APPROVED** [review]

**Pull request merged**